docs/src/pages/utils/transitions/transitions.md

```diff
@@ -8,6 +8,11 @@ Material-UI provides a number of transitions that can be used to introduce some
 [motion](https://material.io/guidelines/motion/material-motion.html)
 to your applications components.
 
+To better support server rendering Material-UI provides a `style` property 
+to the children of some transition components (Fade, Grow, Zoom, Slide).
+The `style` property is to be applied on the child components for the animation to work
+as expected.
```

Comment:
"The `style` property must be applied to the child component for the animation to work as expected."